data/building-hours/1-4-bag-lunch-line.yaml

```diff
@@ -1,9 +1,10 @@
 name: Bag Lunch Line
 category: Food
+links:
+  - {title: 'Bag lunch express order', url: 'https://stolaf.cafebonappetit.com/bag-lunch/'}
 
 schedule:
   - title: Hours
-    notes: Closed until further notice
     hours: []
```

Comment:
I think they published pick-up hours, right?

Reply:
"Pick up in Stav between 2-7pm the day before you need your bag lunch."

and

"Bag lunch will take your lunch swipe for the days that you have requested"

Reply:
Right, so do we need hours for walk ups? And then hours for 24 hour in advance order pickup?

Reply:
I vote we delete the bag lunch hours. It feels too specialized now.